docs/apache-airflow/administration-and-deployment/logging-monitoring/callbacks.rst

```diff
@@ -29,6 +29,12 @@ For example, you may wish to alert when certain tasks have failed, or have the l
     As such, task changes set by the command line interface (:doc:`CLI <../../howto/usage-cli>`) or user interface (:doc:`UI <../../ui>`) do not
     execute callback functions.
 
+.. warning::
```

Comment:
Should this level be a warning or note?

Reply:
i put it as warning because it wasn't obvious that there wouldn't be any errors shown in ui when callback function is broken (like function calls were incorrect syntax)

Reply:
warning is fine. There are some static checks/docs build errors to fix though (see the failed checks).